src/index.html

```diff
@@ -8,6 +8,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="wrapper">
+      <div class="stars stars--0">
+        <div class="stars__star star__noActive star_noMargin"></div>
```

Comment:
You should manage active stars with modifier for `stars` BEM block
You should not add any additional classes to the stars element